.github/workflows/chart.yaml

```diff
@@ -0,0 +1,26 @@
+name: publish_helm_chart
+
+on:
+  push:
+    branches:
+      - main
+      - master
+    paths:
+      - ".github/workflows/chart.yaml"
+      - "charts/**"    
+
+jobs:
+  deploy:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+          fetch-depth: 0
+      - name: Publish Helm chart
+        uses: stefanprodan/helm-gh-pages@v1.4.1
```

Comment:
is there a way to pin to SHAs? I'd hate for this to get compromised and then publish bad charts!

Reply:
Pinned it to `f1701eb82e4d4b82016e7965501c8b6d79feaec9` and added a comment